src/lib.rs

```diff
@@ -78,14 +78,12 @@
 
 #[macro_use]
 extern crate log;
-extern crate serde;
 #[macro_use]
 extern crate serde_derive;
```

Comment:
Furthermore, we don't need to rely on `serde_derive` directly anymore. Instead, in `Cargo.toml`, use:

    serde = { version = "x", features = ["derive"] }

Reply:
Fixed.